src/components/app/App.jsx

```diff
@@ -12,6 +13,7 @@ function App() {
   return (
     <AppProvider store={store}>
       <Switch>
+        <Route path="/" exact component={NotFoundPage} />
         <Route path="/:enterpriseSlug" component={DashboardPage} />
```

Comment:
Should we have a catch-all route defined at the bottom? If we do, is the route for the root path (i.e., "/") necessary?

Example: `<Route path="*" component={NotFoundPage}`